### DISCUSSION TITLE
Node target blank

### DISCUSSION THREAD

**Discussion by JarettWorks** [SnowdogApps/magento2-menu#311] [Q&A]
I'm currently running Snowdog on my M2 project, but I want two of my nav links to open in new tabs/windows - shouldn't the Node target blank checkbox make this happen? It's not working on my side. I've flushed every cache, incognito mode, etc. - still opens in the same window. 

![image](https://github.com/SnowdogApps/magento2-menu/assets/136187485/c908db15-306d-4953-af8f-ffb2e6632d34)

**Comment by andrzejborkowski** [comment]
hi @JarettWorks,

yes, this is how it should work for node type=Custom URL. I checked it with our latest menu module version https://github.com/SnowdogApps/magento2-menu/tree/2.25.0 and it works as intented, you can check it on https://menu-hyva.snowdog.dev/ -> Women -> Tops -> Custom URL Target Blank node. As you can see this link is opened in new tab.

Regards,
Andrzej
<img width="967" alt="Screenshot 2024-01-23 at 09 51 55" src="https://github.com/SnowdogApps/magento2-menu/assets/73829518/e2bb86cf-3890-4dcb-94a2-321bcf17c5ee">

**Reply by manikbhardwajbentley** [reply]
Hi,

it is not working. i used this in M2 and try to create menu with following details.

![image](https://github.com/SnowdogApps/magento2-menu/assets/142475825/d47ddfa0-0d13-4244-b664-4b4c91b496a1)
 it is showing blank on frontend.
![image](https://github.com/SnowdogApps/magento2-menu/assets/142475825/700c9778-3253-4c66-90ee-df1159d23f81)

**Comment by andrzejborkowski** [comment]
hi @bentleycommerce ,
on your screens, we can see that you use Luma theme along with *hyva-topmenu-desktop* identifier that was created for Hyva theme. We don't provide ready to use menu for Luma theme. You can find more detailed information here https://github.com/SnowdogApps/magento2-menu/wiki/Setup#important-note-about-styling

**Reply by manikbhardwajbentley** [reply]
Thanks. Its working now.
